src/basic/math/poly_sin.s

```diff
@@ -0,0 +1,23 @@
+;; #LAYOUT# STD *       #TAKE
+;; #LAYOUT# X16 BASIC_0 #TAKE-OFFSET 2000
+;; #LAYOUT# *  BASIC_0  #TAKE
+;; #LAYOUT# *   *       #IGNORE
+
+; Polynomial for the sine function
+
+
+poly_sin:
```

Comment:
Shouldn’t the polynomial coefficients be placed at the original address?

Reply:
Ok, I'll check if anyone has documented this address.

Reply:
I cannot find any documentation on this address. Do you think it would matter for interoperability?

Reply:
Seems to be $E2EF, I have found it on a German Wiki: https://www.c64-wiki.de/wiki/KERNAL

But you are right, this is probably not important for interoperability, plus the BASIC area starting from $E000 is very precious. I’ll merge this as is.